integration-tests/jms-artemis-client/src/main/resources/application.properties

```diff
@@ -15,9 +15,13 @@
 ## limitations under the License.
 ## ---------------------------------------------------------------------------
 #
+
+quarkus.artemis.devservices.extra-args=--no-autotune --mapped --no-fsync --java-options=-Dbrokerconfig.maxDiskUsage=-1
```

Comment:
Nice work! where did you get this `brokerconfig.maxDiskUsage`? It shoud fix https://github.com/apache/camel-quarkus/issues/5760 as well?

Reply:
I got it from here:

https://github.com/apache/activemq-artemis/blob/18e6f1a88d2c8770c199cf32df3389a05e4f507f/docs/user-manual/configuration-index.adoc#system-properties